### PR TITLE
Base table component

### PR DESCRIPTION
**Pull request by mkanoria**
[WIP]
The base table seems to be working, need to abstract out some more logic into components

Code is pretty terrible rn, just a PoC

### CODE REVIEW

File: frontend/src/components/Table.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useTable } from 'react-table';
```
Comment (OustanDing) [inline frontend/src/components/Table.js:2]: I think we want to put comments on every import

**Review by OustanDing — APPROVED** [review]
LGTM. Just a small comment.

File: frontend/src/components/Table.js (hunk 1 above)
Comment (mkanoria) [reply frontend/src/components/Table.js:2]: Thanks, will update!

**Comment by mkanoria** [comment]
Closing since we moved this over on staging

**Pull request closed without merging**